triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceMain.scala

```diff
@@ -216,7 +224,9 @@ object Server {
     implicit val esf: ExecutionSequencerFactory =
       new AkkaExecutionSequencerPool("TriggerService")(untypedSystem)
 
-    var triggers: Map[UUID, ActorRef[TriggerActor.Message]] = Map.empty
+    var triggers: Map[UUID, TriggerActorWithParty] = Map.empty
+    var triggersByParty: Map[String, Set[UUID]] = Map.empty
```

Comment:
There is a `Party` newtype in `com.daml.ledger.api.refinements`

Reply:
Thanks, will look into it in a follow on PR